app/html/app.js

```diff
@@ -56,6 +59,29 @@ exports.create = function (api) {
       addError(ev.error || ev)
     })
 
+    ////// TODO - extract this to keep patch-lite isolated from electron
+    const { getCurrentWebContents, getCurrentWindow } = electron.remote
+    window.addEventListener('resize', () => {
+      var wc = getCurrentWebContents()
+      wc && wc.getZoomFactor((zf) => {
+        api.settings.sync.set({
+          electron: {
+            zoomFactor: zf,
+            windowBounds: getCurrentWindow().getBounds()
+          }
+        })
+      })
+    })
+
+    var zoomFactor = api.settings.sync.get('electron.zoomFactor')
+    if (zoomFactor)
+      getCurrentWebContents().setZoomFactor(zoomFactor)
+
+    var bounds = api.settings.sync.get('electron.windowBounds')
```

Comment:
here's what getting the settings out looks like (I think I mentioned in docs that it follows lodash/get style)